<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>微语 - ByteDesk Qt Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>会话列表</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="threadListWidget">
          <property name="toolTip">
           <string>会话列表</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QLabel" name="chatTitleLabel">
          <property name="text">
           <string>聊天窗口</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="chatTextBrowser">
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
          <property name="openLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="inputLayout">
          <item>
           <widget class="QLineEdit" name="messageLineEdit">
            <property name="placeholderText">
             <string>输入消息...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="actionRefreshThreads"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="separator"/>
   <addaction name="actionRefreshThreads"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>登录到服务器</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>登出</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionRefreshThreads">
   <property name="text">
    <string>刷新会话</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
